io.github.zen_browser.zen.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.zen_browser.zen
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.mozilla.firefox.BaseApp
+base-version: '23.08'
+command: launch-script.sh
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pcsc
+  - --socket=cups
+  - --persist=.zen
+  - --filesystem=xdg-download:rw
+  - --device=all
+  - --talk-name=org.freedesktop.FileManager1
+  - --system-talk-name=org.freedesktop.NetworkManager
+  - --talk-name=org.a11y.Bus
+modules:
+  - name: zen_browser
+    buildsystem: simple
+    build-commands:
+      - mv zen/* /app/
+
+      - install -d /app/share/icons
+
+      - cp -r icons/* /app/share/icons
+      - install -Dm0755 launch-script.sh ${FLATPAK_DEST}/bin/launch-script.sh
+      - install -Dm0644 policies.json /app/bin/distribution/policies.json
+
+      - install -Dm0644 io.github.zen_browser.zen.metainfo.xml ${FLATPAK_DEST}/share/metainfo/${FLATPAK_ID}.metainfo.xml
+      - install -Dm0644 io.github.zen_browser.zen.desktop ${FLATPAK_DEST}/share/applications/${FLATPAK_ID}.desktop
+
+    sources:
+      - type: archive
+        url: https://github.com/zen-browser/desktop/releases/download/1.0.0-a.7/zen.linux.tar.bz2
```

Comment:
as previously said, this is missing a checksum. (it's not optional)

Reply:
Where do I get the checksums?

Reply:
Calculate it, using `sha256sum $FILE`